uisdk/src/test/java/tech/dojo/pay/uisdk/presentation/ui/result/mapper/PaymentResultViewEntityMapperTest.kt

```diff
@@ -22,7 +22,7 @@ class PaymentResultViewEntityMapperTest {
             .willReturn("Successful Payment")
         val expected = PaymentResultState.SuccessfulResult(
             appBarTitle = "Successful Payment",
-            imageId = 2131230882,
+            imageId = 2131230888,
```

Comment:
thing to update here in the future, we should never use static ids to check values in test

Reply:
will include this as part of data flow refactor  PR